### ISSUE TITLE
contact request

### ISSUE THREAD

**Issue by Bra1nsen** [cimdokjr/Shell.ai-Level---1#1]
are you still working on this project? i have a skycam and a good data set. maybe we can work together on the evaluation.

**Comment by cimdokjr** [comment]
No, we have stopped working. Our model didn't provide sufficient results.
